### PR TITLE
Removes Marines from all Jobs except for a handful.

### PR DESCRIPTION
**Pull request by GrayRachnid**
- Removed Marines from Bridge officers. Reason: The ship is ran by the NTEF and Marine Officers aren't trained to navigate a ship. They're marines.
- ~Removed Marine XO. Reason: The XO is meant to be the second in command that takes care of berthing and also makes sure all personnel are kept in check. It makes no sense to have a Marine XO on a NTEF ship.~
- ~Removed marine command staff. Reason: The command structure of an NTEF ship should only be compromised of NTEF personnel.~
- Removed all marines from medical. Reason: Medical is the responsibility of NTEF and usually marines in any fiction or real life don't have their own doctors.
- ~Remove all marines from engineering. Reason: I actually do not have a valid argument for this. I'm willing to put them back if people find reason in the other removals.~
- ~Remove all marines from supply. Reason: Let NTEF run the logistics, not marines. Can maybe keep deck tech marines but the DO definitely has to stay NTEF.~
- ~Remove Marine BC. Reason: As the CoS's second in command, he should definitely be NTEF.~
- Remove Marines from Cooks. Reason: NTEF runs the ship, they should run accomodations.

As for what I kept.
- Kept Marine SEA. Reason: Will make a future PR, separate and regardless of this, to replace him with a Marine Officer Attache (O1/O2).
- Kept Marine MPs. Reason: You have marines on board, you'll need some MPs to keep the peace. This also will satisfy some people so.
- Kept Marine Chaplains. Reason: It's a chaplain.
- Kept Marine Janitors and Crewmen. Reason: NJP Duty.
- Kept Marine Infantry. Duh.

===================================================

Edit: After talks with maintainer, kept what was permitted.

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
Shitstorm incoming

**Comment by GrayRachnid** [comment]
> 
> 
> Shitstorm incoming

If you want to say something at least say something constructive (:

**Comment by Sun-Soaked** [comment]
I'm just warning you. If you want my genuine opinion on the top, there's no real reason to remove modular variability between different rounds based on people's specific char. Just because these roles -can- be marines doesn't mean they all will be. There are a few particularly prickly ones (Marine XO lmao) but other then that it's not really something I think it's worth upsetting people over.

**Comment by SovietCyanide** [comment]
It might not be realistic but Marines are pretty heavily ingrained currently and a lot of really good characters are marines. It may not be 100% realistic, but I'd like to keep Marines because it may not be realistic but it's fun. And it's such an abrupt change and pretty disliked from what I've seen.

**Comment by GrayRachnid** [comment]
> 
> 
> I'm just warning you. If you want my genuine opinion on the top, there's no real reason to remove modular variability between different rounds based on people's specific char. Just because these roles -can- be marines doesn't mean they all will be. There are a few particularly prickly ones (Marine XO lmao) but other then that it's not really something I think it's worth upsetting people over.

It just never made sense to me that there's so much Marine presence on a ship that's primarily meant to be NTEF. Okay I get it they're supposed to be attached but why do they take up so many roles that are meant to be ran by NTEF? A Marine XO makes no sense. Marine bridge officers don't make sense either, they're not trained to navigate ships. 
Maybe I can see them being enlisted helping out departments, but I cannot see marines taking up command positions anywhere realistically.

**Comment by Tristan63** [comment]
Imagine the hassle of changing your records and the likes when an entire branch is removed from most roles onboard the ship. If the community as it stands wanted this, we wouldn't see people playing Marines as much as they do.

**Comment by GrayRachnid** [comment]
> 
> 
> It might not be realistic but Marines are pretty heavily ingrained currently and a lot of really good characters are marines. It may not be 100% realistic, but I'd like to keep Marines because it may not be realistic but it's fun. And it's such an abrupt change and pretty disliked from what I've seen.

The only standing argument is that a lot of people have marine characters but let me add that a lot of people had to port said characters. When you go to a server without Marines, do you go to their github and demand for marines to be added? No. You just make a new character in whatever branches or roles the server allows and you play the game.
I know that sounds harsh but that's just my opinion.

**Comment by InfinitelyThinRectangles** [comment]
I...don't get this.

This is going to create an enormous amount of upset from the playerbase for no tangible reward that I can see. It's a prefix on someone's ID and whether someone's part of the NTEF or the Sol Marines just doesn't come up as an issue with any frequency that I've seen.

I don't understand what problem this exists to solve, and it's going to make lots of people (justifiably IMHO) really quite spectacularly mad. Why does this exist, what's it an attempt to fix?

**Comment by GrayRachnid** [comment]
> I don't understand what problem this exists to solve, and it's going to make lots of people (justifiably IMHO) really quite spectacularly mad. Why does this exist, what's it an attempt to fix?

It's an attempt to make a HRP environment more realistic when it comes to the lore of who runs the ship.

**Comment by Sun-Soaked** [comment]
Perhaps we should alter the setting to accommodate it, if it doesn't make sense- a large number of people enjoy the status quo as it is, and would be upset if it was altered to prevent them from playing their current roles. Considering that we're currently engaged in hostile-ifying the overall enviroment (more offship threats, dangerous-er away sites, canonically cold-war approaching hot-war setting) it does somewhat make sense for the vessel to have more of a direct military presence.

**Comment by BreadBoy420** [comment]
Gray, I love you babe, I really do and I gotta respect the balls it took for 'yah to make a PR like this, cause it's fucking ballsy. However, I gotta disagree with this move. I understand that you personally want this, however it is not something the community wants. A good portion of Vesta's population plays a Marine character, the neutering of the SGMC is gonna upset a lot of people, and potentially lead to a lot of individuals losing interest in the server, which leads to a dip in our population. I haven't even heard a lot of people complain about the SGMC's presence anyway, this just doesn't seem like something the vast majority of the community wants.

**Comment by GrayRachnid** [comment]
> 
> 
> Perhaps we should alter the setting to accommodate it, if it doesn't make sense- a large number of people enjoy the status quo as it is, and would be upset if it was altered to prevent them from playing their current roles. Considering that we're currently engaged in hostile-ifying the overall enviroment (more offship threats, dangerous-er away sites, canonically cold-war approaching hot-war setting) it does somewhat make sense for the vessel to have more of a direct military presence.

A more direct military presence means more Infantry, not marines in positions that involve running the ship.

**Comment by InfinitelyThinRectangles** [comment]
Do you genuinely believe that fixing some easily handwaved issues with contractual weirdness between NT and the solgov marines is worth creating several barneys, two kerfuffles, a small cluster of brouhahas and at least one full on riot? I really don't see the evidence for the urgency on this.

For real though. This isn't worth the player upset it'll cause. At the very least this should be community consultation, not someone getting out of bed one day and deciding that I WILL REWRITE THE BACKGROUND IN WHICH SEVERAL HUNDRED PEOPLE'S CHARACTERS EXIST TODAY!

**Comment by Jackofashepard** [comment]
image

**Comment by Jackofashepard** [comment]
What does everyone think of the stuff in green

**Comment by Tristan63** [comment]
> What does everyone think of the stuff in green

Just deny the whole thing I say

**Comment by InfinitelyThinRectangles** [comment]
I'll again repeat that I think this NEEDS to be community consultation, not just someone waking up one morning and setting out to fix the server according to their vision. Speaking as server staff, we exist to create and curate an environment _our players_ want to play in, not what _we_ personally prefer.

Just my two cents.

**Comment by ghost** [comment]
> What does everyone think of the stuff in green

Id still keep marine cooks, other than that, it isnt actually all that bad imo

**Comment by GrayRachnid** [comment]
> 
> 
> > What does everyone think of the stuff in green
> 
> Id still keep marine cooks, other than that, it isnt actually all that bad imo

It's an NTEF ship, let NTEF run accommodations

**Comment by nooodontdothatyouretoopretty** [comment]
Not really in favor of any of this. I'd much rather just keep it how it is currently.

**Comment by GrayRachnid** [comment]
Considering we're in a High roleplay environment that sticks close to their lore, our lore is currently incompatible with the roles that the marines get. 

The Marines are meant to be nothing but an attached unit that make sure the dagon is safe, they're not meant to take up all these roles and especially not command positions.

I can see marine enlisted working in engineering, security and supply but I can't imagine why you'd ever have SMC Command Officers in a situation that does not even give them any authority. The lore for the Dagon as it is does not encourage this.

**Comment by Jaraci** [comment]
A funny meme argument bait PR littered with non-justifications for stuff such as:

> Remove all marines from engineering. Reason: I actually do not have a valid argument for this. I'm willing to put them back if people find reason in the other removals.

and:

> Kept Marine Chaplains. Reason: It's a chaplain.

These aren't justifications -- these are you throwing whatever first comes to mind into the PR summary in lieu of any actual explanation because you can't even be bothered to put effort into trying to rile people up.

The only reason this PR exists is because Gray thought it would be funny to poke the hornets' nest and watch people (rightfully?) kick off about one man throwing his funny staff weight around threatening to dig up the ground their characters were built on. This is reprehensible behaviour in anyone's book, but I look forward to watching Gray slam into reverse gear as fast as he can and start backpedalling now that he's started the shitstorm he evidently wanted to set light in the first place.

You don't actually give a shit about the lore and IC justifications for this, otherwise any level of actual effort would show in your reasoning in the PR summary. You just want to start shit, and unfortunately you'll probably manage to get away with it. Sad!

**Comment by GrayRachnid** [comment]
> The only reason this PR exists is because Gray thought it would be funny to poke the hornets' nest and watch people (rightfully?) kick off about one man throwing his funny staff weight around threatening to dig up the ground their characters were built on. This is reprehensible behaviour in anyone's book, but I look forward to watching Gray slam into reverse gear as fast as he can and start backpedalling now that he's started the shitstorm he evidently wanted to set light in the first place.

I am not poking a hornet's nest. I am very serious about this PR. I did not have a summary in place because the lore does it for me. I'm sorry if you think this is me deliberately aggravating people.

**Comment by InfinitelyThinRectangles** [comment]
I'm gonna throw my cards down on the table here but frankly I don't really give a damn about which fictional branch of a fictional military our fictional spaceship is run by.

My objection here is that this change as initially presented results in a large number of players suddenly having characters that they've invested emotional energy into being non-viable, literally overnight.

That's a really shitty thing to do, man. C'mon, I _cannot_ be the only one who can see that?

**Comment by io-util** [comment]
> Removed all marines from medical. Reason: Medical is the responsibility of NTEF and usually marines in any fiction or real life don't have their own doctors.

Probably the funniest thing I have heard today, I don't know how exactly it works in America, I give you that, but here in Germany our Armed Forces also have their own hospitals and doctors that are employed by them specifially.

**Comment by GrayRachnid** [comment]
> 
> 
> I'm gonna throw my cards down on the table here but frankly I don't really give a damn about which fictional branch of a fictional military our fictional spaceship is run by.
> 
> My objection here is that this change as initially presented results in a large number of players suddenly having characters that they've invested emotional energy into being non-viable, literally overnight.
> 
> That's a really shitty thing to do, man. C'mon, I _cannot_ be the only one who can see that?

I see where you're coming from and I already addressed it in a previous message.
> The only standing argument is that a lot of people have marine characters but let me add that a lot of people had to port said characters. When you go to a server without Marines, do you go to their github and demand for marines to be added? No. You just make a new character in whatever branches or roles the server allows and you play the game.

Alternatively, you just make changes to your character to accommodate the new changes.

**Comment by Jaraci** [comment]
I have no doubt that you are very serious about this PR. I do have doubts as to why you are serious about it.

Do I believe for half a second you give a flying fuck about lore, roleplay, and/or gameplay integrity? Not even for that long.
Do I believe you threw this up as fast as you could with zero effort to QC it or even half-assedly attempt to tie in whatever changes you felt like making with any semblance of lore compliance, gameplay considerations, or community sentiment? You bet I do.

I don't buy your excuses for a second, but I can see you're already reversing as fast as you can without out-and-out closing the PR, so please don't let me get in the way of your damage control for too long.

**Comment by Karl-Franz-I** [comment]
> > Removed all marines from medical. Reason: Medical is the responsibility of NTEF and usually marines in any fiction or real life don't have their own doctors.
> 
> Probably the funniest thing I have heard today, I don't know how exactly it works in America, I give you that, but here in Germany our Armed Forces also have their own hospitals and doctors that are employed by them specifially.

Your army has an army hospital on top of a navy ship with a navy hospital? Interesting. It's an NT ship, staffed by their personnel.

File: maps/torch/job/torch_jobs_boh.dm [hunk 1]
```diff
@@ -33,14 +33,11 @@
 
 /datum/job/cmo
 	allowed_branches = list(
-		/datum/mil_branch/fleet,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/command/cmo/marine
```
Comment (io-util) [inline maps/torch/job/torch_jobs_boh.dm:37]: I see no reason to remove the CMO from being a marine, military does indeed have own doctors in some countries, actual doctors in Military Hospitals. That goes for the entirety of medbay.

**Comment by SovietCyanide** [comment]
So considering Grey didn't read the lore and there really is no reason for this change, is this going to continue?

File: maps/torch/job/torch_jobs_boh.dm [hunk 2]
```diff
@@ -189,59 +183,45 @@
 /datum/job/senior_doctor
 	allowed_branches = list(
 		/datum/mil_branch/fleet,
-		/datum/mil_branch/civilian = /decl/hierarchy/outfit/job/torch/crew/medical/contractor/senior,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/medical/senior/marine
```
Comment (io-util) [inline maps/torch/job/torch_jobs_boh.dm:193]: Same reasoning as above.

File: maps/torch/job/torch_jobs_boh.dm [hunk 3]
```diff
@@ -189,59 +183,45 @@
...
-		/datum/mil_rank/marine_corps/o3,
-		/datum/mil_rank/marine_corps/o4,
 		/datum/mil_rank/civ/contractor
 	)
 
 /datum/job/doctor
 	allowed_branches = list(
 		/datum/mil_branch/fleet,
-		/datum/mil_branch/civilian = /decl/hierarchy/outfit/job/torch/crew/medical/contractor,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/medical/doctor/marine
```
Comment (io-util) [inline maps/torch/job/torch_jobs_boh.dm:207]: Also the same as above.

File: maps/torch/job/torch_jobs_boh.dm [hunk 4]
```diff
@@ -189,59 +183,45 @@
...
-		/datum/mil_rank/marine_corps/e4,
-		/datum/mil_rank/marine_corps/e5,
-		/datum/mil_rank/marine_corps/e6,
 		/datum/mil_rank/civ/contractor
 	)
 
 /datum/job/medical_trainee
 	allowed_branches = list(
-		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/medical/doctor/fleet,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/medical/doctor/marine
```
Comment (io-util) [inline maps/torch/job/torch_jobs_boh.dm:224]: This too.

File: maps/torch/job/torch_jobs_boh.dm [hunk 5]
```diff
@@ -189,59 +183,45 @@
...
-		/datum/mil_rank/marine_corps/e2,
-		/datum/mil_rank/marine_corps/e3
+		/datum/mil_rank/fleet/e3
 	)
 
 /datum/job/psychiatrist
 	allowed_branches = list(
 		/datum/mil_branch/civilian,
-		/datum/mil_branch/fleet = /decl/hierarchy/outfit/job/torch/crew/medical/counselor/ec,
-		/datum/mil_branch/marine_corps = /decl/hierarchy/outfit/job/torch/crew/medical/counselor/marine
```
Comment (io-util) [inline maps/torch/job/torch_jobs_boh.dm:237]: And especially this, Military / Marines definitely have their own psychiatrists to help soldiers cope with things they see.

**Review by io-util — CHANGES_REQUESTED** [review]
I'm just gonna judge the medical part of this - which I absolutely cannot understand. Besides that, I am not gonna mix myself into this _very constructive argument_ too much.

**Comment by Tristan63** [comment]
This whole PR is justifying how the customer is wrong when infact the customer is always right.

**Comment by GrayRachnid** [comment]
> 
> 
> I have no doubt that you are very serious about this PR. I do have doubts as to why you are serious about it.
> 
> Do I believe for half a second you give a flying fuck about lore, roleplay, and/or gameplay integrity? Not even for that long.
> Do I believe you threw this up as fast as you could with zero effort to QC it or even half-assedly attempt to tie in whatever changes you felt like making with any semblance of lore compliance, gameplay considerations, or community sentiment? You bet I do.
> 
> I don't buy your excuses for a second, but I can see you're already reversing as fast as you can without out-and-out closing the PR, so please don't let me get in the way of your damage control for too long.

I'm only accommodating requests from the maintainers because they're the ones who decide if this is merged at the end. I don't see how that's reversing. I'm not going to close the PR but the maintainers have already decided it's not going to get merged. I asked them to keep it open just to facilitate discussion. At this point you're not even discussing anything constructive and instead personally attacking me and my reason for making this PR. What are you even attempting to achieve at this point? 

If you want to discuss how to make it better then i'm all for it. I've already had a few changes of heart after discussions with people. They are as following.
Bridge officers can stay but the Marine Attache would need to outrank them.(however I still think bridge staff should be NTEF)
XO should definitely not be a marine.
I'm still against Marine department heads.
Medical Marines can stay since the SMC is the only armed force of SG(means it has doctors)

At the end of the day I think that marines should only be taking up enlisted positions in every department and they'd only have one officer on board.

**Comment by GrayRachnid** [comment]
> 
> 
> So considering Grey didn't read the lore and there really is no reason for this change, is this going to continue?

Which part of the lore?

**Comment by SovietCyanide** [comment]
> > So considering Grey didn't read the lore and there really is no reason for this change, is this going to continue?
> 
> Which part of the lore?

You know what part you literally told us in Discord that you didn't even read it.

**Comment by GrayRachnid** [comment]
> 
> 
> I'm just gonna judge the medical part of this - which I absolutely cannot understand. Besides that, I am not gonna mix myself into this _very constructive argument_ too much.

Yeah i'm sorry for that I had thought that marines wouldn't be doctors based on the Fiction and RL references of Marines but turns out our lore only has Marines as the SG armed force which means it'd have doctors. I'm not going to make any more changes to the PR since it was already denied, i'm only having it kept open to facilitate discussion and see if we can take this in a direction that actually accommodates the lore.

**Comment by GrayRachnid** [comment]
> 
> 
> > > So considering Grey didn't read the lore and there really is no reason for this change, is this going to continue?
> > 
> > 
> > Which part of the lore?
> 
> You know what part you literally told us in Discord that you didn't even read it.


I misread, it's a mistake, I apologized, admitted to it but it still didn't actually change my standing argument. The mistake I made is irrelevant to the overarching reason that this PR was made.

**Comment by Jaraci** [comment]
> At this point you're not even discussing anything constructive and instead personally attacking me and my reason for making this PR. What are you even attempting to achieve at this point?

What I am doing is highlighting that the PR was opened without any intention of being constructive and undermining this belated attempt to spin it around as simply a big "oops!" mistake as if you were somehow misguided all along. I'm sorry if this is detracting from you trying to paint this whole PR as having been created and pushed in good faith right from the word go.

"I'm sorry, I made a mistake" is like #1 in the playbook as the go-to fallback plan after deliberately and knowingly going all-in to provoke as much of a reaction as you can. You'll forgive me for calling BS.

**Comment by GrayRachnid** [comment]
> "I'm sorry, I made a mistake" is like #1 in the playbook as the go-to fallback plan after deliberately and knowingly going all-in to provoke as much of a reaction as you can. You'll forgive me for calling BS.

I made a mistake in assuming that the SMC was a Corps falling under the SCG fleet rather than the whole armed unit of the SCG.
That's the only thing i'm apologizing for.

**Comment by Stutternov** [comment]
Weighing in with my two cents; I’d completely agree with removing Marine XO due to the issues in the past of ‘this is sol ship now’ - it makes little sense in that way to have second in command being someone who is not NTEF controlling, representing and ordering an NTEF vessel. It would be like an army general seizing control of vessel because ‘hewwo, my rank??? this is mine now’

I even understand engineering since it has caused weird things in the past. 

However removing SMC from medical jobs such as Corpsman I’m NOT in support of. Only compromise on that would be adding the corpsman to infantry. And I know that would still be heavily unpopular.

While I’d normally support a PR like this I honestly don’t suggest it at all. A change like this should have been made before the server went live and planned out. People have been playing the server and this would make them toss their characters out the window. 

Little too late in my opinion for a change like this.

**Comment by io-util** [comment]
I don't see any reason to keep this open since it is not gonna be merged anyway, the most stuff here is just "personal attacks" or continuing a really dumb argument anyway. If you have something important and constructive to say and did not have the opportunity to say it, go to #dev-public on our Discord, if it's gonna be some low effort meme comments, don't even think about it.

**Pull request closed without merging**